tests/test-sources/plugins/utils/which-key.nix

```diff
@@ -8,46 +8,54 @@
       enable = true;
 
       # Testing for registrations
-      registrations."f" = {
-        prefix = "<leader>";
-        mode = [
-          "n"
-          "v"
-          "i"
-          "t"
-          "c"
-          "x"
-          "s"
-          "o"
-        ];
-        name = "Group Test";
-        f = "Label Test";
-        "1" = [
-          {
-            __raw = ''
-              function()
-                print("Raw Lua Code and List KeyMapping Test")
-              end
-            '';
-          }
-          "Raw Lua Code and List KeyMapping Test"
-        ];
-        "oo" = "Label Test 2";
-        "<tab>" = {
-          name = "Group in Group Test";
-          f = [
-            {
-              __raw = ''
-                function()
-                  vim.cmd("echo 'Raw Lua Code and List KeyMapping Test 2'")
-                end
-              '';
-            }
+      settings.spec = [
+        {
+          __unkeyed-1 = "f";
 
-            "Raw Lua Code and List KeyMapping Test 2"
+          prefix = "<leader>";
```

Comment:
Also, doesn't seem like prefix is supported anymore.. when I create a group of mappings using a prefix with the mappings, they dont appear to group. If I include the prefix in each of the unkeyed attributes then they group properly.

Reply:
Yeah, looks like its deprecated, too. https://github.com/folke/which-key.nvim/blob/6c1584eb76b55629702716995cca4ae2798a9cca/lua/which-key/mappings.lua#L51-L54

Reply:
Example of what I'm talking about:
Using prefix: 
<img width="368" alt="image" src="https://github.com/user-attachments/assets/08963125-3bd6-4c42-a672-a86b9458a80b">
Using unkeyed with same combination in a single string:
<img width="388" alt="image" src="https://github.com/user-attachments/assets/791e71da-df34-4315-b45d-636842143b07">

Reply:
Looks like `prefix` used to prefix the `lhs` (key) for every mapping defined in the tree, however I think now it is intended to be done explicitly for each mapping.

This makes sense as the new format is no longer a tree.